.build/docker/.env.base

```diff
@@ -1,11 +1,11 @@
-VITE_APP_VERSION=0.74.0
-VITE_APP_NAME=@alkemio/client-web
-VITE_APP_BUILD_VERSION=dev
-VITE_APP_BUILD_DATE=dev
-VITE_APP_BUILD_REVISION=dev
-VITE_APP_ALKEMIO_DOMAIN=http://localhost:3000
-VITE_APP_SENTRY_AUTH_TOKEN=
-VITE_APP_LOG_ERRORS=true
-VITE_APP_COLLAB_URL=http://localhost:4002
-VITE_APP_COLLAB_PATH=/socket.io
-VITE_APP_GRAPHQL_ENDPOINT=/graphql
+VITE_APP_VERSION           = 0.75.2
```

Comment:
Let's exclude this change. This file is in the .gitignore, and I'm not sure where you're getting this formatting.

Reply:
It's coming from an extension which I just removed from my IDE. If this file is in the `.gitignore` file how come git detects it as a change?